openfisca_core/enumerations.py

```diff
@@ -6,6 +6,7 @@ def __init__(self, varlist, start = 0):
         self._vars = {}
         self._nums = {}
         self._count = 0
+        self.list = varlist
```

Comment:
Good catch 💯 !

Indeed, if users send an invalid value, we must return a `400` response (and not a `500`), and give them a clear answer so that they can fix their request.
As it's a client mistake, and not an internal server error, we don't need to log, but we need to catch the error.